znoyder/config.d/43-override-OSP-18.yml

```diff
@@ -626,6 +626,7 @@ override:
         vars:
           extra_commands:
             - dnf install -y python3-placement python3-placement-tests python3-testresources python3-testscenarios
+            - cp -r {{ zuul.project.src_dir }} /tmp/python-osc-placement && pip install /tmp/python-osc-placement
```

Comment:
Could you please add a comment that this is needed due to https://bugzilla.redhat.com/show_bug.cgi?id=2238782